### DISCUSSION TITLE
Download service – hybrid implementation - example

### DISCUSSION THREAD

**Discussion by VeronikaKusova** [INSPIRE-MIF/helpdesk#128] [Implementation examples]
Dear all,
Do you know any example for so called „hybrid implementation“ of INSPIRE download service? It joins ATOM and WFS.

Best regards.
Veronika

**Comment by pawelsoczewski** [comment]
Hi Veronika, 
In Poland we published some download services as „hybrid implementation“. In Poland most of download services are published as „hybrid implementation“. Here is Catalog of Environmental monitoring dataset, you can find some examples: [https://inspire.gios.gov.pl/geonetwork](url)

**Comment by VeronikaKusova** [comment]
Hi Pawel,
Thank you for your quick response. I have gone through the catalogue. And for example is this service https://inspire.gios.gov.pl/geonetwork/srv/eng/catalog.search#/metadata/bb21ac32-b483-47c8-8524-b709214c1c31 the hybrid one? I am not sure if I understand it correctly but does “hybrid service” mean that ATOM and WFS are just linked through feed/subtitle? So there are not any additional differences from “normal” WFS and ATOM?
